src/components/IntroFeatures.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardHeader from '@material-ui/core/CardHeader';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  layout: {
+    width: 'auto',
+    marginLeft: theme.spacing.unit * 3,
+    marginRight: theme.spacing.unit * 3,
+    [theme.breakpoints.up(900 + theme.spacing.unit * 3 * 2)]: {
+      width: 900,
+      marginLeft: 'auto',
+      marginRight: 'auto',
+    },
+  },
+  heroContent: {
+    maxWidth: 600,
+    margin: '0 auto',
+    padding: `${theme.spacing.unit * 8}px 0 ${theme.spacing.unit * 6}px`,
+  },
+  cardHeader: {
+    backgroundColor: theme.palette.grey[200],
+  },
+  cardIntroFeatures: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'baseline',
+    marginBottom: theme.spacing.unit * 2,
+  }
+});
+
+class IntroFeatures extends Component{
+  render() {
+    const { classes } = this.props;
+    return (    
+      <React.Fragment>
+        <CssBaseline />
```

Comment:
Holycow, now you scared me... Which documentation did I copied from :(

Reply:
By the way, shouldn't this CssBaseLine go to App instead? Since App has theme-specific detail, but index.js does not necessarily have that detail yet?